README.md

```diff
@@ -32,3 +32,18 @@ Run API unit tests with `make api-unit-test`
 Run API integration tests with `make api-integration-test`
 
 This command will spin up a postgres container, run the API integration tests, then close the created container.
+
+## Database Backup restore
+
+Database backups occur on every deployment and also during the scheduled backup window.
+
+To restore the database form a backup the following steps need to performed in the specified order
+
+* Find the snapshot to restore from the AWS console 
+* snapshots created during a build are tagged with the commit sha
+* Uncomment everything from the file `terraform/db_backup.tf`
+* Update local var `snapshot_name` to the snapshot name from the console
+* Uncomment the line `POSTGRES_HOST     = aws_rds_cluster.pgsql_backup.endpoint` from `terraform/api.tf`
+* Comment out the line  `POSTGRES_HOST     = aws_rds_cluster.pgsql.endpoint` from `terraform/api.tf`
+* Run `make plan` and `make apply`
+* This should create a new rds cluster from the snapshot provided and update api to point to the new backup cluster
```

Comment:
At the end of this activity we have 2 clusters running. 

What is the process steps to delete / remove the second cluster (old) 

How does this affect other environments and how can we keep the other envs consistent?

Reply:
* Added steps to destroy existing cluster
* Added commands so any changes are scoped to the ENV we need this on.